app/services/moderator/manage_activity_and_roles.rb

```diff
@@ -78,6 +78,14 @@ def handle_user_status(role, note)
         check_super_admin
         remove_negative_roles
         user.add_role :super_admin
+      when "Tech Admin"
+        check_super_admin
+        remove_negative_roles
+        user.add_role :tech_admin
```

Comment:
**nitpick (non-blocking):** Ok, this is a super nitpick, but we're calling `add_role` twice in a row and use different code styles for the parenthesis around arguments. The general consensus is to use them unless it's a DSL method like `belongs_to`, so maybe we can add them and make my OCD happy?

Reply:
> The general consensus is to use them unless it's a DSL method like belongs_to
@citizen428 I was not aware of this, but thanks for the heads up :) 

The reason I did it like this was because glancing at the rest of the methods (and the file) it seemed that the pattern was when [we have one parameter for add_role we do not use parenthesis](https://github.com/forem/forem/blob/1a2dc2ec51d849a28e2ee9f9bdaca8ced440da6c/app/services/moderator/manage_activity_and_roles.rb#L71), but [when we have two parameters then we do](https://github.com/forem/forem/blob/1a2dc2ec51d849a28e2ee9f9bdaca8ced440da6c/app/services/moderator/manage_activity_and_roles.rb#L92). 

It doesn't make sense to just add the parenthesis to this line, if this is the general consensus, we should probably add it to the rest of the file too which I can do if that's the preference? I also think it would beneficial to reference a good style guide, or perhaps Rubocop should have been complaining about this.

Reply:
These are some of the related cops. The tl;dr is use parenthesis unless it's an internal or framework DSL or when the method call has no arguments.

https://github.com/rubocop-hq/ruby-style-guide#method-definition-parentheses
https://github.com/rubocop-hq/ruby-style-guide#dsl-method-calls
https://github.com/rubocop-hq/ruby-style-guide#method-calls-with-no-arguments
https://github.com/rubocop-hq/ruby-style-guide#methods-that-are-part-of-an-internal-dsl

Reply:
Thanks @citizen428, I'll update this file in a subsequent PR 👍